src/Nri/Colors.elm

```diff
@@ -172,32 +172,32 @@ blue =
 
 {-|
 
-<p style="font-size:2em; color: #00488A">#00488A</p>
+<p style="font-size:2em; color: #0e5595">#0e5595</p>
 
 -}
 blueDark : Css.Color
 blueDark =
-    hex "#00488A"
+    hex "#0e5595"
```

Comment:
💭 old colors should just be equal to the new name, rather than duplicating the hex value  `blueDark = navy`